tests/integration/README.md

```diff
@@ -223,6 +223,73 @@ To assert that two TSV files must be equal, wrap them in the `TSV` class and use
 statement. Example: `assert TSV(result) == TSV(reference)`. In case the assertion fails, `pytest`
 will automagically detect the types of variables and only the small diff of two files is printed.
 
+### Debug mode
+
+Here is how you run a debugger for the server under integration test:
+
+1. Put the **_statically linked binary_** of your debugger into the ClickHouse root folder of your repo. The [nnd](https://github.com/al13n321/nnd) debugger is ideal for this purpose (and for ClickHouse debugging in general).
```

Comment:
It looks like currently **only** nnd is supported by the script.

It's OK, but let's make it clear.


```suggestion
1. Put the **_statically linked binary_** of your debugger into the ClickHouse root folder of your repo. The [nnd](https://github.com/al13n321/nnd) debugger is ideal for this purpose (and for ClickHouse debugging in general), it's currently the only supported option.
```